cmd/lakectl/cmd/cherry_pick.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"net/http"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api"
+	"github.com/treeverse/lakefs/pkg/uri"
+)
+
+const (
+	cherryPickCmdArgs = 2
+)
+
+// lakectl cherry-pick lakefs://myrepo/main lakefs://myrepo/some-ref
+var cherryPick = &cobra.Command{
+	Use:   "cherry-pick <branch uri> <ref uri>",
```

Comment:
This is backwards from merge:
```go
var mergeCmd = &cobra.Command{
	Use:   "merge <source ref> <destination ref>",
	Short: "Merge & commit changes from source branch into destination branch",
	Long:  "Merge & commit changes from source branch into destination branch",
```

I think we should switch the order.  Maybe also keep the "source"/"destination" terminology.  It is perhaps clearer which is which.